stacks-signer/src/signerdb.rs

```diff
@@ -617,6 +613,18 @@ impl SignerDb {
         try_deserialize(result)
     }
 
+    /// Return the last accepted block the signer (highest stacks height). It will tie break a match based on which was more recently signed.
+    pub fn get_signer_last_accepted_block(&self) -> Result<Option<BlockInfo>, DBError> {
+        let query = "SELECT block_info FROM blocks WHERE json_extract(block_info, '$.state') IN (?1, ?2) ORDER BY stacks_height DESC, json_extract(block_info, '$.signed_group') DESC, json_extract(block_info, '$.signed_self') DESC LIMIT 1";
```

Comment:
Are these in (or will soon be) new columns, instead of a JSON blob?

Reply:
Yes :) Its in the tenure extend PR though so didn't want to add conlficts here and an unnecessary db schema version change. Will have to update this code in the tenure extend PR if it gets merged first or vice versa.